_sections/30-bindings.md

```diff
@@ -2,7 +2,7 @@
 title: Bindings
 ---
 
-A `<bind>` element wires together a primary instance node and the presentation of the corresponding question to the user. It is used to describe the datatype and various kinds of logic related to the data. A bind can refer to any node in the primary instance including repeated nodes. It may or may not have a corresponding presentation node in the [body](#body). 
```

Comment:
Looks like this (and a few other things) are whitespace corrections. Happy to revert them if they seem problematic, but they look right to me.

Reply:
I think these kinds of changes make it harder to follow the history and for a spec that might be more important. My preference would be to revert them.

Reply:
Yeah, very good point that it could be a big deal for a spec.

Reply:
Actually I do think it's worth correcting these whitespace errors as they are a problem for Atom with default settings. What I'm going to do is fix introduce a commit before my changes that just adds the whitespace changes, so it's clear in the history that's the change. Let me know if that still feels like a bad idea!